packages/babel/README.md

```diff
@@ -174,7 +174,7 @@ export default {
   ],
   output: [
     { file: 'bundle.cjs.js', format: 'cjs' },
-    { file: 'bundle.esm.js', format: 'esm' }
+    { file: 'bundle.esm.js', format: 'es' }
```

Comment:
nit: if the preferred value of the `format` has changed from `esm` to `es`, should we also reflect that in the `output.file` presented here?

Reply:
Fair enough